lib/kontentClient.ts

```diff
@@ -151,6 +151,19 @@ export const getArticlesForListing = (usePreview: boolean, page?: number, pageSi
       .then(res => res.data);
   }
 
+  export const getArticlesCount = (usePreview: boolean) => deliveryClient
```

Comment:
Great! I might make it a bit more general - getItemCount with optional type attribute -> to be able to showcase and reuse it (and show people how to get items count).